client/src/components/ProjectForm.js

```diff
@@ -109,52 +48,118 @@ const StyledRadio = styled(Radio)(({ theme }) => ({
+  const [locationType, setLocationType] = useState('remote');
+  // State to track the toggling from Project view to Edit Project View via edit icon.
+  const [editMode, setEditMode] = useState(true);
   const { auth } = useAuth();
-  const { register, handleSubmit, formState: { errors } } = useForm({ 
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
     mode: 'all',
     defaultValues: {
       name: '',
       description: '',
       location: '',
       githubUrl: '',
       slackUrl: '',
-      googleDriveUrl: ''
-    }
+      googleDriveUrl: '',
+    },
   });
 
-  const routeToNewProjectPage = () => {
-     if(newlyCreatedID !== null) {
-      history.push(`/projects/${newlyCreatedID}`)
-    }
-  }
-  
-  useEffect(() => {
-    routeToNewProjectPage()
-  },[newlyCreatedID])
+  // ----------------- Submit requests -----------------
 
-  // only handles radio button change
-  const handleRadioChange = (event) => {
-    setLocationType(event.target.value);
+  // Handles POST request found in api/ProjectApiService.
+  const submitNewProject = async (data) => {
+    const projectApi = new ProjectApiService();
+    try {
+      const id = await projectApi.create(data);
+      history.push(`/projects/${id}`);
+    } catch (errors) {
+      console.error(errors);
+      return;
+    }
   };
 
-  const submitForm = async (data) => {
+  // Fires PUT request to update the project,
+  const submitEditProject = async (data) => {
     const projectApi = new ProjectApiService();
     try {
-      const id = await projectApi.create(data);
-      setNewlyCreatedID(id);
+      await projectApi.updateProject(projectToEdit._id, data);
     } catch (errors) {
       console.error(errors);
       return;
     }
-    setActiveButton('close');
+    setEditMode(true);
   };
 
+  // ----------------- Handles and Toggles -----------------
+
+  // Handles the location radio button change.
+  const handleRadioChange = (event) => {
+    setLocationType(event.target.value);
+  };
+  // Toggles the project view to edit mode change.
+  const handleEditMode = (event) => {
+    setEditMode(!editMode);
+  };
+
+  // ----------------- Icons -----------------
+
+  // Holds the Add New Project Icon and styling.
+  const addIcon = () => {
+    return (
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          cursor: 'pointer',
+        }}
+      >
+        <PlusIcon style={{ p: 1 }} />
+        <Typography sx={{ p: 1, fontSize: '14px', fontWeight: '600' }}>
+          Add New Project
+        </Typography>
+      </Box>
+    );
+  };
+  // Holds the Edit New Project Icon and styling.
+  const editIcon = () => {
+    return (
+      <Box
+        sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}
+      >
+        <EditIcon
+          style={{ p: 1, cursor: 'pointer' }}
+          onClick={handleEditMode}
+        />
```

Comment:
Changes included but need to revisit. 

Issue: Adding a cancel button needs to reset input values back to its original state. Currently with this change, if a user makes a change, then cancels, the users changes are still in the input values. They however will not be submitted unless save is clicked. Would adding 'Close' or 'Exit editing' be a good temporary fix?

Reply:
I feel like restoring original state isn't a big task, can you add that to this PR?

Reply:
I can look into it but I think we should invest in a new method when requesting/handling extra features. The original commit did not have some of these features because they're not included in the Figma v.4 version. I know the Website team implemented ERs - Emergent Request which are meant to keep production going and to prevent delays/blockers. ERs are issues that are created for out of scope tasks that build off the original issue created.

Reply:
I hear you on that. The funcationality as it stands I think can be confusing to the user. They might think if they press the Edit button, then chagne things, then Edit again the project has been edited/updated.